ffi/dotnet-idsdk/IoticsIdLib/src/Identity.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System.Net;
+
+namespace IOTICS;
+
+// Custom RuntimeException class
+public class IoticsLibException(string message) : ApplicationException(message)
+{
+}
+
+public class DelegationException(string message) : IoticsLibException(message) {
+    
```

Comment:
(pedantic) above one has no newline between braces. - to they even need the `{}` (or single line)

Reply:
done